_api-reference/cat/cat-aliases.md

```diff
@@ -16,20 +16,32 @@ has_children: false
 The CAT aliases operation lists the mapping of aliases to indexes, plus routing and filtering information.
 
 
-## Path and HTTP methods
-
+<!-- spec_insert_start
+api: cat.aliases
+component: paths_and_http_methods
+-->
+## Paths and HTTP methods
 ```json
-GET _cat/aliases/<alias>
-GET _cat/aliases
+GET /_cat/aliases
+GET /_cat/aliases/{name}
 ```
-{% include copy-curl.html %}
+<!-- spec_insert_end -->
 
+<!-- spec_insert_start
+api: cat.aliases
+component: query_parameters
+-->
 ## Query parameters
-
 Parameter | Type | Description
 :--- | :--- | :---
-local | Boolean | Whether to return information from the local node only instead of from the cluster manager node. Default is `false`.
-expand_wildcards | Enum | Expands wildcard expressions to concrete indexes. Combine multiple values with commas. Supported values are `all`, `open`, `closed`, `hidden`, and `none`. Default is `open`.
+`expand_wildcards` | String / String / String / String / List | Whether to expand wildcard expression to concrete indexes that are open, closed or both.
```

Comment:
Why are there 4 instances of String in the type? Also, as a rule, we don't want slashes to signify "or". I would suggest "String or list". Also, the old description had supported values, which is helpful, and specified that the list should be comma-separated. The new description has less information.

Reply:
@nhtruong: IS there something in the spec that would cause this?

Reply:
That's coming form the spec. I'll update the code to handle this edge case.

Reply:
Sounds good. Let me know once that update is through and I'll update this PR with the updated descriptions.